sqlobject/src/main/java/org/jdbi/v3/sqlobject/BatchHandler.java

```diff
@@ -142,9 +142,8 @@ public boolean hasNext() {
                 }
                 // execute a single chunk and buffer
                 PreparedBatch batch = handle.prepareBatch(sql);
-                applyCustomizers(batch, args);
                 for (int i = 0; i < chunkSize && batchArgs.hasNext(); i++) {
-                    applyBinders(batch.add(), batchArgs.next());
+                    applyCustomizers(batch.add(), batchArgs.next());
```

Comment:
Note that this changes means that _all_ method parameters are now batch-able to a `@SqlBatch` method--so users may have to tack on a `@SingleValue` for things like `@BindList` or `@DefineList`.